src/main/java/fr/ospiea/oscra/employee/controller/EmployeeController.java

```diff
@@ -0,0 +1,47 @@
+package fr.ospiea.oscra.employee.controller;
+
+import fr.ospiea.oscra.employee.object.Employee;
+import fr.ospiea.oscra.employee.service.EmployeeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * Created by fqin on 18/09/2016.
+ */
+@RestController
+@RequestMapping("/api/employees")
```

Comment:
Here, I add '/api' because I want to distinguish the front-end and back-end. When the proxy receives requests, the proxy searches url-mapping either in front-end, or in back-end. It could not search url-mapping in both sides. Maybe some configurations of proxy or using balance loader could solve this problem, but at this moment, for me, adding '/api' is the simplest solution.